vllm/model_executor/layers/quantization/__init__.py

```diff
@@ -16,6 +18,7 @@
     "gptq": GPTQConfig,
     "squeezellm": SqueezeLLMConfig,
     "marlin": MarlinConfig,
+    "sparseml": CompressedTensorsConfig
```

Comment:
why `sparseml` and not `compressed-tensors`?

Reply:
To comply with the vllm config parsing. The quantization method listed in the sparsmel model config is `sparseml`

Reply:
Can you coordinate with Sara on this?

I think it should be `compressed-tensors` in the HF config

Reply:
I'm fine with the update, but it is currently called "sparseml" in the compressed-tensors repo so it will need to be updated there too